contracts/base/FunctionHandlerManager.sol

```diff
@@ -69,6 +69,8 @@ abstract contract FunctionHandlerManager is RegistryManager {
             revert FunctionHandlerNotSet(account, functionSelector);
         }
 
+        checkPermittedModule(functionHandler);
```

Comment:
Should revert or skip if flagged?

Reply:
In my opinion, since this doesn't affect the core execution flow, it's OK to revert here.